rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -265,6 +275,15 @@ public Integer receiveHeader(BtcBlock header) throws IOException, BlockStoreExce
         return 0;
     }
 
+    private boolean cannotProcessNextBlock(StoredBlock previousBlock) {
+        int nextBlockHeight = previousBlock.getHeight() + 1;
+        boolean networkIsMainnet = btcContext.getParams().equals(NetworkParameters.fromID(NetworkParameters.ID_MAINNET));
+
+        return nextBlockHeight >= bridgeConstants.getBlockWithTooMuchChainWorkHeight()
+            && networkIsMainnet
```

Comment:
With the Integer.MAX constant for other networks this becomes unnecessary, right?

Reply:
technically yes, but we decided to check the network anyways for transparency